src/parsers/transactions.ts

```diff
@@ -148,6 +148,7 @@ export const convertNewTransactionToRainbowTransaction = (tx: NewTransaction): R
     hash: tx.hash,
     nonce: tx.nonce,
     protocol: tx.protocol,
+    timestamp: Date.now(),
```

Comment:
this function is used from both the "add new txn" flow and the "udpate txn" flow (for speed up / cancel)